monkeytype/stubs.py

```diff
@@ -239,6 +245,7 @@ def get_updated_definition(
     func: Callable,
     traces: Iterable[CallTrace],
     rewriter: Optional[TypeRewriter] = None,
+    ignore_existing_annotations: bool = False
```

Comment:
trailing comma

Reply:
This is the last line, do we need to add a comma?

```
def get_updated_definition(
    func: Callable,
    traces: Iterable[CallTrace],
    rewriter: Optional[TypeRewriter] = None,
    ignore_existing_annotations: bool = False
) -> FunctionDefinition:
```

Reply:
Yes, our preference is always for trailing comma on every line of a multi-line list. It's not necessary for Python's sake, but it makes future additions or reorderings of the list much easier, with less diff noise.